coalib/settings/Setting.py

```diff
@@ -112,20 +136,23 @@ def __init__(self,
         self.key = key
         self.origin = str(origin)
 
-    def __path__(self, origin=None):
+    def __path__(self, origin=None, glob_escape_origin=False):
         """
         Determines the path of this setting.
 
         Note: You can also use this function on strings, in that case the
         origin argument will be taken in every case.
 
-        :param origin:      the origin file to take if no origin is specified
-                            for the given setting. If you want to provide a
-                            directory, make sure it ends with a directory
-                            seperator.
-        :return:            An absolute path.
-        :raises ValueError: If no origin is specified in the setting nor the
-                            given origin parameter.
+        :param origin:             the origin file to take if no origin is
```

Comment:
You start with small letter and no period here, but the next param has capital and period.

We dont have a style guide for this yet - but could you use the same guide in the same function for consistency ?